server/middlewares/errors/errorHandler.ts

```diff
@@ -0,0 +1,21 @@
+/* eslint-disable no-unused-vars */
+import {
+  Request, Response, NextFunction,
+} from 'express';
+import CustomError from '../../utils/CustomError';
```

Comment:
Import CustomError from the utils(I mean from the index not from the CustomError)